### ISSUE TITLE
Enable Content Title to be Keyboard Accessible 

### ISSUE THREAD

**Issue by lauragift21** [lauragift21/giftegwuenu.dev#34]
According to A11y your page to should be accessible to keyboard only users. Make the content accessible such that when I tab over any content I should be able to access it.

**Comment by lauragift21** [comment]
This is already resolved
